frontend/app-development/features/textEditor/TextEditor.tsx

```diff
@@ -46,18 +39,12 @@ useEffect(() => {
   } = useTextResourcesQuery(org, app);
 
   const setSearchQuery = (search: string) => {
-    const params: any = { lang: searchParams.get('lang') };
+    const params: any = {};
     if (search.length > 0) {
       params.search = search;
     }
     setSearchParams(params);
```

Comment:
what about making this a single line? Then we do not need to use any type ethier.
```suggestion
    setSearchParams(search.length > 0 ? { search } : {});
```

Reply:
Done!